.github/workflows/test-dev.yml

```
@@ -57,11 +57,12 @@ jobs:
           files: build/test-results/test/*.xml
 
       - name: 테스트 커버리지 업로드
-        uses: codecov/codecov-action@v4
+        uses: codecov/codecov-action@v5
         if: >
           github.event_name == 'push' ||
           github.event.pull_request.head.repo.full_name == github.repository
         with:
           token: ${{secrets.CODECOV_TOKEN}}
+          use_gpg: false
```

actionlint (1.7.12)

[error] input "use_gpg" is not defined in action "codecov/codecov-action@v5". available inputs are "base_sha", "binary", "codecov_yml_path", "commit_parent", "directory", "disable_file_fixes", "disable_safe_directory", "disable_search", "disable_telem", "dry_run", "env_vars", "exclude", "fail_ci_if_error", "files", "flags", "force", "gcov_args", "gcov_executable", "gcov_ignore", "gcov_include", "git_service", "handle_no_reports_found", "job_code", "name", "network_filter", "network_prefix", "os", "override_branch", "override_build", "override_build_url", "override_commit", "override_pr", "plugins", "recurse_submodules", "report_code", "report_type", "root_dir", "run_command", "skip_validation", "slug", "swift_project", "token", "url", "use_legacy_upload_endpoint", "use_oidc", "use_pypi", "verbose", "version", "working-directory"

(action)